ako-infra/avirest/netinfo_handler.go

```diff
@@ -113,10 +113,10 @@ func (t *T1LRNetworking) SyncLSLRNetwork() {
 		return
 	}
 
-	if len(lslrmap) > 0 {
-		dataNetworkTier1Lrs, updateRequired := t.removeStaleLRLSEntries(client, cloudModel, lslrmap)
-		if !t.matchSegmentInCloud(dataNetworkTier1Lrs, lslrmap) || updateRequired {
-			cloudModel.NsxtConfiguration.DataNetworkConfig.Tier1SegmentConfig.Manual.Tier1Lrs = t.constructLsLrInCloud(dataNetworkTier1Lrs, lslrmap)
+	if len(lrlsMap) > 0 {
+		dataNetworkTier1Lrs, updateRequired := t.removeStaleLRLSEntries(client, cloudModel, lrlsMap)
+		if !t.matchSegmentInCloud(dataNetworkTier1Lrs, lrlsMap) || updateRequired {
```

Comment:
@abhishekbsingh as `updateRequired` already evaluated, can we make it as first condition in `if condition` so that evaluation is faster?

Reply:
done